mxml.mod/mxml/mxml-file.c

```diff
@@ -1529,7 +1529,7 @@ mxml_stream_write(_mxml_streambuf_t *buf)	/* I - File descriptor buffer */
   int		bytes;			/* Bytes written */
   unsigned char	*ptr;			/* Pointer into buffer */
 
-  
+
```

Comment:
Some clients allow you to selectively commit only the interesting code changes.
This limits noise and distraction from the stuff that's actually fixing the broken things.

Reply:
I thought this was a single line and always wondered why people do not clean their sources (btw the whole source file has pretty odd indentations).

I will try to remember to selectively commit stuff the next time.

Reply:
You are right - after your comment on this line I was already aware of the "3rd party - diff" issue this can create.

Maybe pushing back our changes to mxml could get accepted there (including the removed whitespace ;-)) - albeit I doubt it.